book/src/chap02.asciidoc

```diff
@@ -195,7 +195,7 @@ Cuando una expresión contiene más de un operador, el orden de evaluación depe
 
 * Los operadores con la misma precedencia se evalúan de izquierda a derecha (excepto potencias). Entonces, en la expresión +grados/2*π+, la división ocurre primero y el resultado se multiplica por +π+. Para dividir entre latexmath:[2\pi], puede usar paréntesis, escribir +grados/2/π+ o +grados/2π+.
 
-[OBSERVACION]
+[OBSERVACIÓN]
```

Comment:
(traducción de google)  El texto "[OBSERVACIÓN]" aquí debe mantenerse como "[TIP]", que es una palabra clave para el software `asciidoctor` y que colocará el texto "[OBSERVACIÓN]" correctamente. Ver #5.  

(english)  The text "[OBSERVACIÓN]" here should actually be "[TIP]", which is a keyword that `asciidoctor` will recognize and (among other things) replace with the text "[OBSERVACIÓN]".  See issue #5.  If you're willing to change this text back to "[TIP]", that would be helpful.

Reply:
Oops, I see... Done in https://github.com/JuliaIntro/IntroAJulia.jl/pull/34/commits/642d48cb28ea8e9a8d72a58bf066a46c2510a267.

Reply:
Thanks!  Not your fault, but thanks for fixing it.